### PR TITLE
#469 - more visual cleanup

### LINKED ISSUE #469: MUI - Visual Cleanup
## Description

before we release the MUI branch, we want to clean up some of the uglier MUI things that will be easy enough to do.

## Acceptance Criteria

- address this doc https://docs.google.com/document/d/1rnsJJaHi7mdU2AJaDAKvL4rw3jZ_NSOkIC35Ijhfklk/edit
- idk if this is in the doc or not but i hate that all the links are purple, they should be the primary NER color

## Proposed Solution
- [x] Light Mode Text Color should be NER dark gray instead of black
- [x] Adjust ner logo in top left to have a bit more spacing on top/bottom, so it looks a little more consistent/ cleaner (see develop logo placement for reference)
- [x] Make player icon in top right bigger
- [x] Make breadcrumbs a little closer together (see develop)
- [x] Outline of trickster mode switch not visible in light mode
- [x]  Make outlines of cards on home page more distinct
- [x] Enable outlines on date range and timeline status in light mode by default
- [x] Space out useful links a bit more
- [x] See if you can make the entirety of Next # Days clickable
- [x] In cards on homepage, make subheaders bold (e.g. End Date, Progress,Engineering Lead, etc.).
- [x] See what we can do with the filter to make it look better (e.g. extend lines down from filters through options) (Projects)
- [x] Project Detail Page: bring info closer to headers and bold headers to make distinct in project details
- [x] Make sure change request link (under changes) is red (not blue) in light mode
- [x] Make less space at bottom of gantt chart
- [x] Add new risk should look like the NER Button (red)/ use the contained button styling.)
- [x] Remove shadow on convert to CR button in risk log
- [x] Make outline between work packages more distinct (light more and dark mode)
- [x] Pills are misaligned with the weeks in work packages - make them smaller so they fit in line
- [x] Project Edit Form: inputs should be outlined by default
- [x] Make delete button look like other delete buttons (red with white trashcan) 
- [x] Submit button should be on right, cancel on left
- [x] Work Package Detail Page: bring info closer to headers and bold headers to make distinct in project details
- [x] Make sure change request link (under changes) is red (not blue) in light mode
- [x] Change Requests Page: See how left aligning the items in each column looks, keep whichever way looks better
- [x] Change Request Detail Page: Make sure change request link next to WBSNum, and under implemented changes is red not blue
- [x] Proposed Solution: Description and Scope impact should be next to each other/ have more space, and budget/timeline impact should be next to each other and have less space
- [x] Info Page: vertical spacing of links isn’t consistent (fix spacing on this page in general)
- [x] Dark Mode Specific: make background dark gray instead of black
- [x] Teams: make sizing more consistent if projects line wraps around (looks weird rn, I feel like the cards should move on to a new line/ extend horizontally rather than changing vertical size when info gets too big)
- [x] Light Mode: Increase contrast between PageBlock/Box elements and Background
- [x] version number? (bottom left of sidebar)
- [x] text inputs should all be outlined correctly by default 💀 
- [x] use form labels instead of just typographies on inputs (make sure for select menus the form labels are supposed to kinda overlap with the outline of the dropdown and this should help reduce the overall vertical height)
- [x] make scrollbars on home page less ugly
- [x] make collapsed page blocks take up less vertical space below the header
- [x] interactive elements placed in the header right of a page block (see home page) should not increase the vertical height of the page block header. using form labels properly should help with this, but if necessary, make the interactive elements be a smaller size
- [x] Fix home page date picker and select labels
- [x] fix Teams cards to be consistent sizes, most likely by simply truncating too many projects

### PR DESCRIPTION
**Pull request by RChandler234**
## Changes

Starting with a couple small things (fixing version number, team summary blocks, and size of collapsed pageblocks). Tried making the default scrollbar for the whole app look cooler, but wasn't having much luck. Gonna keep going through the rest as I have time.

## Screenshots

image
image

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please review the [contributor guide](https://nerdocs.atlassian.net/wiki/spaces/NER/pages/8060929/Software+Contributor+Guide) and reach out to your Tech Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [ ] All commits are tagged with the ticket number
- [ ] No linting errors / newline at end of file warnings
- [ ] All code follows repository-configured prettier formatting
- [ ] No merge conflicts
- [ ] All checks passing
- [ ] Screenshots of UI changes (see Screenshots section)
- [ ] Remove any non-applicable sections of this template
- [ ] Assign the PR to yourself
- [ ] No `yarn.lock` changes (unless dependencies have changed)
- [ ] Request reviewers & ping on Slack
- [ ] PR is linked to the ticket (fill in the closes line below)

Closes #469  (issue #)

### CODE REVIEW

File: src/frontend/src/layouts/Sidebar/Sidebar.tsx [hunk 1]
```diff
@@ -49,6 +50,7 @@ const Sidebar: React.FC = () => {
   return (
     <div className={styles.sidebar}>
       <NavPageLinks linkItems={linkItems} />
+      <Typography className={styles.versionNumber}>v.3.5.0</Typography>
```
Comment (anthonybernardi) [inline src/frontend/src/layouts/Sidebar/Sidebar.tsx:53]: this'll be 3.5.1 probably when we release next

**Review by anthonybernardi — CHANGES_REQUESTED** [review]
nice!

**Review by Ari-S-123 — CHANGES_REQUESTED** [review]
image

Can you make the project names not get cutoff somehow.

**Comment by RChandler234** [comment]
> image
> 
> Can you make the project names not get cutoff somehow.

Truncating was the solution to make all the cards the same size, if you can think of another solution I can make the change, but this was the easiest fix I/James could come up with for now

File: src/frontend/src/layouts/Sidebar/Sidebar.tsx [hunk 2]
```diff
@@ -49,6 +50,7 @@ const Sidebar: React.FC = () => {
   return (
     <div className={styles.sidebar}>
       <NavPageLinks linkItems={linkItems} />
+      <Typography className={styles.versionNumber}>v.3.5.1</Typography>
```
Comment (anthonybernardi) [inline src/frontend/src/layouts/Sidebar/Sidebar.tsx:53]: i dont see this on my screen for some reason (also now it'll be 3.5.3 when this gets released probably)

**Review by anthonybernardi — COMMENTED** [review]
amazing. looks so much better. those 8 pixels on the page block honestly make a huge difference

**Review by jamescd18 — APPROVED** [review]
LGTM

**Review by anthonybernardi — APPROVED** [review]
lovely

**Pull request merged**